src/components/CreateAnalysis/FileLookup.tsx

```diff
@@ -1,102 +1,95 @@
         payload: {
-          images: filteredDcmImages
+          studyUID: studyInstances[0].studyInstanceUID
         }
       });
-      const studyInstances: StudyInstance[] = CreateAnalysisService.extractStudyInstances(filteredDcmImages);
-      if (studyInstances.length > 0) {
-        dispatch({
-          type: CreateAnalysisTypes.UpdateCurrSelectedStudyUID,
-          payload: {
-            studyUID: studyInstances[0].studyInstanceUID
-          }
-        });
-      }
-    }, [seriesInstanceUID, minCreationDate, maxCreationDate]);
-
-    const clearFilters = async () => {
-      setSeriesInstanceUID('');
-      setMinCreationDate('');
-      setMaxCreationDate('');
     }
+  }, [minCreationDate, maxCreationDate]);
 
-    const onMinDateChange = (_str: string, date?: Date) => {
-      if (date && isValidDate(date)) {
-        setMinCreationDate(date.toISOString().substring(0, 10));
-        if (date >= new Date(maxCreationDate)) {
-          setMaxCreationDate('');
-        }
+  const clearFilters = async () => {
+    setMinCreationDate('');
+    setMaxCreationDate('');
+  }
+
+  const onMinDateChange = (_str: string, date?: Date) => {
+    if (date && isValidDate(date)) {
+      setMinCreationDate(date.toISOString().substring(0, 10));
+      if (date >= new Date(maxCreationDate)) {
+        setMaxCreationDate('');
       }
-    };
+    }
+  };
 
-    const onMaxDateChange = (_str: string, date?: Date) => {
-      if (date && isValidDate(date)) {
-        setMaxCreationDate(date.toISOString().substring(0, 10));
-        if (date <= new Date(minCreationDate)) {
-          setMinCreationDate('');
-        }
+  const onMaxDateChange = (_str: string, date?: Date) => {
+    if (date && isValidDate(date)) {
+      setMaxCreationDate(date.toISOString().substring(0, 10));
+      if (date <= new Date(minCreationDate)) {
+        setMinCreationDate('');
       }
-    };
+    }
+  };
 
-    return (
-        <React.Fragment>
-        <div className="InputRow">
-          <div className="InputRowField">
-            <label>Series UID</label>
-            <TextInput value={seriesInstanceUID} type="text" onChange={setSeriesInstanceUID} aria-label="Series UID" />
-          </div>
-          <div className="InputRowField">
-            <label>File Creation Date</label>
-            <Split hasGutter>
-              <SplitItem>
-                <DatePicker
-                  value={minCreationDate}
-                  onChange={onMinDateChange}
-                  aria-label="From date"
-                />
-              </SplitItem>
-              <SplitItem>to</SplitItem>
-              <SplitItem>
-                <DatePicker
-                  value={maxCreationDate}
-                  onChange={onMaxDateChange}
-                  aria-label="To date"
-                />
-              </SplitItem>
-            </Split>
-          </div>
+  return (
+    <>
+      <div className="InputRow">
+        <div className="InputRowField">
```

Comment:
I would suggest adding the following child style to `InputRowField` to make the date input boxes match the width of the Clear Filters button:
```
    div {
      justify-content: space-between;
    }
```

![Screenshot from 2021-04-13 09-14-14](https://user-images.githubusercontent.com/24356368/114558556-cc9ee880-9c38-11eb-850a-3cfb74c6feb3.png)
to
![Screenshot from 2021-04-13 09-13-53](https://user-images.githubusercontent.com/24356368/114558589-d6285080-9c38-11eb-9be3-2a5fbe1c9ebf.png)